public/templates/navBar.html

```diff
@@ -0,0 +1,82 @@
+<!-- Additional CSS Files -->
+<meta charset="utf-8" />
+<meta
+  name="viewport"
+  content="width=device-width, initial-scale=1, shrink-to-fit=no"
+/>
+<link
+rel="stylesheet"
+type="text/css"
+href="/assets/css/bootstrap.min.css"
+/>
+<link
+rel="stylesheet"
+type="text/css"
+href="/assets/css/font-awesome.css"
+/>
+<link
+rel="stylesheet"
+type="text/css"
+href="/assets/css/templatemo-art-factory.css"
+/>
```

Comment:
Can we double check that all these imports are needed for the header? From inspection, I only see templatemo-art-factory.css and bootstrap.min.css being applied

Reply:
Every page uses these imports differently. Some pages may use some and others won't. If we're going to generalize everything it may be easier to keep them all than create some kind of conditional checking for every page.